api/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/frankban/quicktest v1.13.0 // indirect
 	github.com/go-test/deep v1.0.2
 	github.com/hashicorp/errwrap v1.1.0
-	github.com/hashicorp/go-cleanhttp v0.5.1
+	github.com/hashicorp/go-cleanhttp v0.5.2
```

Comment:
I'm not sure why but this was causing one of the preflight checks to fail because the go mod download automatically upgraded to the latest patch version (0.5.2) and then the next check was finding that the go.mod file was modified and failed. So I just updated it and tests started passing!?

Reply:
Looks like we're on 0.5.2 on main already, so I guess you weren't alone.